Core/Page Object/Base.ts

```diff
@@ -0,0 +1,22 @@
+import {Locator, Page} from '@playwright/test';
+import {Dashboards} from "./dashbord";
+import {LoginPage} from "./loginPage";
+
+export class BasePage {
+  readonly page: Page;
+  readonly dashboardPage: Dashboards;
+  readonly loginPage: LoginPage;
+  readonly sideBarMenu:Locator
+
+  constructor(page: Page) {
+    this.page = page
+    this.loginPage = new LoginPage(this.page);
+    this.dashboardPage = new Dashboards(this.page);
+    this.sideBarMenu = page.locator('aside')
```

Comment:
this.sideBarMenu is a component that has to be extracted into separate class and added only to pages which contain it